pkg/dao/types/status/status_resource.go

```diff
@@ -58,5 +58,11 @@ var resourceStatusPaths = NewWalker(
 
 // WalkResource walks the given status by resource flow.
 func WalkResource(st *Status) *Summary {
-	return resourceStatusPaths.Walk(st)
+	summary := resourceStatusPaths.Walk(st)
```

Comment:
It's not limited to resource summary. How do we declare components like completed jobs/pods are inactive?
<img width="1585" alt="image" src="https://github.com/seal-io/walrus/assets/5697937/a2166c27-6245-4442-8d25-64c4b188c46b">

Reply:
Updated. Changed the Decide function to return a `Summary` directly, which encapsulates inactive status.